common/src/tx/mod.rs

```diff
@@ -224,6 +221,32 @@ impl Tx {
     }
 }
 
+#[derive(Debug, Clone, RlpEncodable, RlpDecodable, Serialize, Deserialize, Getters)]
+#[getset(get = "pub")]
+pub struct TxTimestamp {
+    tx_hash: TxHash,
+    timestamp: u64,
+}
+
+impl TxTimestamp {
+    pub fn new(tx_hash: TxHash, timestamp: u64) -> Self {
+        Self { tx_hash, timestamp }
+    }
+}
```

Comment:
I think this `TxTimestamp` does not have any specific usage.
Is there any special use case I don't know?

I think this one can be merged with `TxSequence`.
What is the reason that we have separate types?

Reply:
Yes, I guess they could be merged together.
Timestamp of TXs is definitely needed for many use-cases